cogs/tickets.py

```diff
@@ -156,21 +164,61 @@ def topic_check(m):
             icon_url=av,
             text="IsThicc Tickets"
         )
-        await ticket.send(content="<@!796953153010532362>", embed=opened)
+        await ticket.send(content="<@&796953153010532362>", embed=opened)
 
         await self.db.execute('INSERT INTO tickets VALUES(' + str(ticket.id) + ', ' + str(reaction_user.id) + ', true)')
-        # parse as str in query, but the database will parse it as whatever you chose it to be in phpmyadmin.
 
-        #   # Channel id(ticket.id):                 ticket_id
-        #   # Ticket opener id(reaction_user.id):    user_id
-        #   # Open or closed(bool: true):            open_close
+    # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+
+    @commands.command()
+    async def add(self, ctx, member: discord.Member):
+        av = self.bot.user.avatar_url
+
+        if not ctx.guild.get_role(796953153010532362) in ctx.author.roles:
```

Comment:
The ticket owner, and staff should be able to add people. I suggest making a function that you can pass the current context to that will tell you if the command executor has permission to execute commands in that ticket. Use the DB + Discord perms to keep track of who can.